modules/@apostrophecms/login/index.js

```diff
@@ -909,8 +915,20 @@ module.exports = {
             last: true
           }
         );
-      }
+      },
 
+      getSchema() {
+        return self.apos.user.schema
+          .filter(({ name }) => [ 'username', 'password' ].includes(name))
+          .map(field => ({
+            name: field.name,
+            label: field.label,
+            placeholder: field.placeholder ?? self.options.placeholder[field.name],
```

Comment:
Why the check for an existing placeholder from the user module? We definitely want one on the login form, so I think you can simplify.

Reply:
It feels confusing to be able to override the label from the user module but not the placeholder, yeah? Or am I confusing something?